app/models/survey.rb

```diff
@@ -7,11 +7,14 @@
 #  description                          :text
 #  created_at                           :datetime         not null
 #  updated_at                           :datetime         not null
+#  first_question_id                    :integer
```

Comment:
Nit: Can we move this up maybe 2nd or 3rd from the top, because it's more important than `created_at` and `updated_at`